docs/using-data-types.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[ecs-data-types]]
+=== Data Types
```

Comment:
ECS is an Elasticsearch schema, and it's all about "data types" throughout. Giving this page the title "Data Types" makes it sound like it's going to be general information about data types in ECS, which is not the case.

I think this page should be named something that's clearly about OSS, so that most users know it doesn't concern them. Since it has to fit in the sidebar it has to be short, so it's hard to come up with a good one :-)

Some ideas:

* ECS in OSS clusters
* ECS with OSS clusters
* Basic datatypes in OSS clusters

Reply:
Great point about the placement of the document. Totally agree that it's appropriate for this to live under 'Additional Information' as it doesn't apply to the majority of users. 

Would field type make sense instead of data type (it's used in the subscription page).

Some ideas:
- ECS Considerations in OSS Clusters
- Licensed field types in OSS clusters 

Maybe we could add a link to the [subscriptions page ](https://www.elastic.co/subscriptions) as it shows wildcard is not available in OSS. Constant keywords isn't listed there, but I can look at getting it added.

Reply:
++ on "field type" wording

I think "ECS Considerations in OSS Clusters" works great, if it fits in the sidebar.

Not sure how multiline headings look in the sidebar, actually. If multiline works, I think "ECS Considerations in OSS Clusters" is my fav.

Reply:
++ good suggestions! I've made the update to `ECS Considerations in OSS Clusters`, and I think it looks acceptable in the sidebar.

Reply:
Happy to see this section title fits well in the sidebar.

Looking at it though, perhaps we should remove the leading "ECS"? I think it's a bit redundant. Perhaps something like "Considerations for OSS Clusters"... WDYT?

Reply:
++ I made this change.